spec/lib/seed/patient_seeder_spec.rb

```diff
@@ -28,6 +28,7 @@
     end
     expect(facility.assigned_patients.count).to eq(4)
     expect(facility.assigned_patients.with_hypertension.count).to eq(4)
-    expect(facility.assigned_patients.with_hypertension.where(status: "active").count).to be >= 3
+    # This is a loose expectation because we introduce randomness into our patient statuses
```

Comment:
Thanks for explaining that, because it does seem odd. I believe I've seen this fail in CI before. Perhaps that's why you lowered the number.

Reply:
> Thanks for explaining that, because it does seem odd. I believe I've seen this fail in CI before. Perhaps that's why you lowered the number.

Yup, that is why. The alternative would be to hack in some deterministic sampling for test mode, but I trust this code enough and don't think its worth it.